src/components/ToolMenu/Draw/Attributions/index.tsx

```diff
@@ -0,0 +1,216 @@
+import React, {
+  useEffect, useState
+} from 'react';
+
+import {
+  MinusCircleOutlined, PlusOutlined
+} from '@ant-design/icons';
+import {
+  Button, Drawer, Form, Divider, Row
+} from 'antd';
+import _cloneDeep from 'lodash/cloneDeep';
+import OlFeature from 'ol/Feature';
+
+import { useTranslation } from 'react-i18next';
+
+import {
+  useMap
+} from '@terrestris/react-geo/dist/Hook/useMap';
+
+import './index.less';
+import AttributionRow from './AttributionRow';
+
+export type AttributionDrawerProps = {
+  openAttributeDrawer: boolean;
+  onClose: (open: boolean) => void;
+};
+
+const AttributionDrawer: React.FC<AttributionDrawerProps> = ({
+  openAttributeDrawer,
+  onClose
+}) => {
+  const [selectedFeature, setSelectedFeature] = useState<OlFeature>();
+  const [render, setRender] = useState<boolean>(true);
+  const [isFormValid, setIsFormIsValid] = useState(true);
+  const [currentProperties, setCurrentProperties] = useState<any>({});
```

Comment:
Very minor, but instead of using `any`, we could use something like `Record<string, any>` or similar.